src/generators/minecraft-axolotl-character/MinecraftAxolotlCharacterGenerator.res

```diff
@@ -0,0 +1,1014 @@
+    standardWidth: 64,
+    standardHeight: 64,
+  },
+  {
+    id: "Gold",
+    url: requireTexture("axolotl_gold"),
+    standardWidth: 64,
+    standardHeight: 64,
+  },
+  {
+    id: "Brown",
+    url: requireTexture("axolotl_wild"),
+    standardWidth: 64,
+    standardHeight: 64,
+  },
+]
+
+let steve = TextureMap.MinecraftCharacterLegacy.steve
+let alex = TextureMap.MinecraftCharacterLegacy.alex
+
+let script = () => {
+  // Define input textures
+  Generator.defineSelectInput("Skin Model Type", ["Steve", "Alex"])
+  Generator.defineTextureInput("Skin", {standardWidth: 64, standardHeight: 64, choices: []})
+  Generator.defineTextureInput(
+    "Tail Fin Texture",
+    {
+      standardWidth: 64,
+      standardHeight: 64,
+      choices: ["Blue", "Cyan", "Pink", "Gold", "Brown"],
+    },
+  )
+  Generator.defineTextureInput(
+    "Head Fins Texture",
+    {
+      standardWidth: 64,
+      standardHeight: 64,
+      choices: ["Blue", "Cyan", "Pink", "Gold", "Brown"],
+    },
+  )
+  // Define user variables
+  Generator.defineBooleanInput("Show Folds", true)
+  Generator.defineBooleanInput("Show Labels", true)
+  Generator.defineBooleanInput("Axolotl Eyes", true)
+  Generator.defineBooleanInput("Show Skin Overlay", true)
+
+  // Get user variable values
+  let alexModel = Generator.getSelectInputValue("Skin Model Type") === "Alex"
+  let showFolds = Generator.getBooleanInputValue("Show Folds")
+  let showLabels = Generator.getBooleanInputValue("Show Labels")
+  let showEyes = Generator.getBooleanInputValue("Axolotl Eyes")
+  let showOverlay = Generator.getBooleanInputValue("Show Skin Overlay")
+
+  // Head
+  Generator.drawTexture(
+    //Axolotl's right
+    "Skin",
+    (0, 8, 8, 8),
+    (189, 214, 40, 40),
+    (),
+  )
+  Generator.drawTexture(
+    //Front
+    "Skin",
+    (8, 8, 8, 8),
+    (229, 214, 64, 40),
+    (),
+  )
+  Generator.drawTexture(
+    //Axolotl's left
+    "Skin",
+    (16, 8, 8, 8),
+    (293, 214, 40, 40),
+    (),
+  )
+  Generator.drawTexture(
+    //Back
+    "Skin",
+    (24, 8, 8, 8),
+    (333, 214, 64, 40),
+    (),
+  )
+  Generator.drawTexture(
+    //Front
+    "Skin",
+    (8, 0, 8, 8),
+    (229, 174, 64, 40),
+    (),
+  )
+  Generator.drawTexture(
+    //Bottom
+    "Skin",
+    (16, 0, 8, 8),
+    (229, 254, 64, 40),
+    ~flip=#Vertical,
+    (),
+  )
+
+  //Body
+  Generator.drawTexture(
+    //Top
+    "Skin",
+    (20, 16, 8, 4),
+    (234, 405, 64, 32),
+    (),
+  )
+  Generator.drawTexture(
+    //Back
+    "Skin",
+    (32, 20, 8, 12),
+    (234, 325, 64, 80),
+    ~flip=#Vertical,
+    (),
+  )
+  Generator.drawTexture(
+    //Bottom
+    "Skin",
+    (28, 16, 8, 4),
+    (378, 405, 64, 32),
+    ~flip=#Vertical,
+    (),
+  )
+  Generator.drawTexture(
+    //Front
+    "Skin",
+    (20, 20, 8, 12),
+    (234, 437, 64, 80),
+    (),
+  )
+  Generator.drawTexture(
+    //Left
+    "Skin",
+    (28, 20, 4, 12),
+    (298, 437, 32, 80),
+    ~rotateLegacy=-90.00,
```

Comment:
The `rotateLegacy` rotates around the top left corner of the texture.

Can you switch to using the non-legacy `rotate` which instead rotates around the center of the texture?